packages/@aws-cdk/aws-lambda-python/lib/packaging.ts

```diff
@@ -46,7 +46,7 @@ export class Packaging {
   public static readonly POETRY = new Packaging({
     dependenciesFile: DependenciesFile.POETRY,
     // Export dependencies with credentials avaiable in the bundling image.
-    exportCommand: `poetry export --with-credentials --format ${DependenciesFile.PIP} --output ${DependenciesFile.PIP}`,
+    exportCommand: `poetry export --without-hashes --with-credentials --format ${DependenciesFile.PIP} --output ${DependenciesFile.PIP}`,
```

Comment:
The only question I have is whether or not this should be an option that the user can opt-in/out.

Reply:
@corymhall new `bundlingOptions.poetryExcludeHashes` member added which leads to the creation of a new `Packaging.POETRY_EXCLUDE_HASHES` object. Let me know what you think of this -- I was trying to leave existing resources unaffected / leave the default behavior as opt **out**.